packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -904,11 +912,15 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
         this.events.on(EventType.MIGRATING, boundPauseOnMigration);
         this.events.on(EventType.ABORTING, boundPauseOnMigration);
 
+        const monitor = this.experiments.monitor ? new Monitor(this.stats, this.log) : null;
```

Comment:
I would keep this conditional, but not inside experiments, we use those for unstable features, I don't forsee any actual problems with it here.